ignite/pkg/xerrors/xerrors.go

```diff
@@ -0,0 +1,56 @@
+// Package xerrors provides helpers for error creation, avoiding
+// using different packages for errors.
+//
+// e.g.:
+// import errors "github.com/ignite/cli/ignite/pkg/xerrors"
+//
+//	func main() {
+//	 err1 := errors.New("error new")
+//	 err2 := errors.Errorf("%s: error", foo)
+//	 err3 := errors.Wrap(errFoo, errBar)
+//	}
+package xerrors
```

Comment:
If we are using this package for dealing with errors we might want to actually use `errors` for the package definition so it can be discovered and automatically imported by typing `errors.`:

```suggestion
package errors
```

Reply:
Also pushed PR #3740 to enforce usage of this package

Reply:
> Can we also add a rule to avoid using `fmt.Errof`?

I'm not sure, I trust it should be possible, I will check but the question is should we avoid it? It would be good to be able to use it in combination with the new `errors` package.